homeassistant/components/philips_js/config_flow.py

```diff
@@ -127,8 +145,9 @@ async def async_step_user(
                 errors["base"] = "unknown"
             else:
                 if serialnumber := hub.system.get("serialnumber"):
-                    await self.async_set_unique_id(serialnumber)
-                    self._abort_if_unique_id_configured()
+                    self._entry = (
+                        await self.async_set_unique_id(serialnumber) or self._entry
+                    )
```

Comment:
I'm not convinced this is correct.
You do not seem to have a test for a user trying to add a duplicate `serialnumber` so it's hard to confirm.

Reply:
I dropped that feature, since as you say there was no explicit test for it.